wpuf-functions.php

```diff
@@ -1518,22 +1518,20 @@ function wpuf_get_currency( $type = '' ) {
  *
  * @return mixed
  */
-function wpuf_format_price( $number, $with_currency = false ) {
-    $number = number_format( (float) $number, 2, '.', '' );
+function wpuf_format_price( $number ) {
 
-    if ( $with_currency ) {
-        $symbol   = wpuf_get_currency( 'symbol' );
-        $position = wpuf_get_option( 'currency_position', 'wpuf_payment', 'left' );
+    $number   = number_format( (float) $number, 2, '.', '' );
```

Comment:
you are converting every number to float with two decimal points, which someone might not want.